### ISSUE TITLE
QT5 new install cannot "Log in to Evernote"

### ISSUE THREAD

**Issue by artmg** [baumgarr/nixnote2#228]
New installation of Lubuntu 16.10 - using the PPA to install Nixnote2. Starts with empty .nixnote folder in home.

```
$ dpkg -s nixnote2|grep Version
Version: 2.0-beta-10-0+800+47+201612021347~ubuntu16.10.1
$ nixnote2
QSocketNotifier: Can only be used with threads started with QThread
DEBUG 2016-12-03 22:01:42.634 ( gui/shortcutkeys.cpp @ 203 ) Loading system shortcuts from  "/usr/share/nixnote2/shortcuts.txt" 
DEBUG 2016-12-03 22:01:42.634 ( gui/shortcutkeys.cpp @ 236 ) Unable to open file for reading or file does not exist. 
DEBUG 2016-12-03 22:01:42.634 ( gui/shortcutkeys.cpp @ 205 ) Loadng user shortcuts from  "/home/art/.nixnote/shortcuts.txt" 
DEBUG 2016-12-03 22:01:42.634 ( gui/shortcutkeys.cpp @ 236 ) Unable to open file for reading or file does not exist. 

(nixnote2:18967): Gtk-WARNING **: Theme directory  imported-Humanity/192 of theme Lubuntu has no size field
```

So the interface starts up here fine, but of course it has not been authorized to use my account yet. Press F9 to synrchonize brings up a small web dialog "Log in to Evernote". After a few seconds waiting the program crashes. Some times some of the green panels of the logon screen appear just before the crash, other times it just stays blank for around 10 seconds before crashing, with the following output on the console...

```
(nixnote2:18967): GLib-GObject-WARNING **: cannot register existing type 'GdkDisplayManager'
(nixnote2:18967): GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed
(nixnote2:18967): GLib-GObject-CRITICAL **: g_object_new: assertion 'G_TYPE_IS_OBJECT (object_type)' failed
Error: signal 11:
nixnote2(_Z13fault_handleri+0x2e)[0x561d9720f9ae]
/lib/x86_64-linux-gnu/libc.so.6(+0x35860)[0x7f19205de860]
/usr/lib/x86_64-linux-gnu/libgdk-3.so.0(gdk_display_manager_get_default_display+0x0)[0x7f18854f94a0]
/usr/lib/mozilla/plugins/libevbrowserplugin.so(NP_GetMIMEDescription+0x30)[0x7f188652e150]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0xa45bf5)[0x7f1922837bf5]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x946e76)[0x7f1922738e76]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x945ba4)[0x7f1922737ba4]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x946357)[0x7f1922738357]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x48ff95)[0x7f1922281f95]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x88199e)[0x7f192267399e]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x860c2e)[0x7f1922652c2e]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x881e3b)[0x7f1922673e3b]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0xf6937d)[0x7f1922d5b37d]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x17298c4)[0x7f192351b8c4]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x172ac81)[0x7f192351cc81]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x19edeeb)[0x7f19237dfeeb]
/usr/lib/x86_64-linux-gnu/libQt5WebKit.so.5(+0x19f7b71)[0x7f19237e9b71]
Segmentation fault (core dumped)
$ 
```
I can easily reproduce this if you require other diagnostics - I just wish I could use nixnote on my new box :(

**Comment by artmg** [comment]
Searching for that error I found a QT_STYLE_OVERRIDE variable, and I managed to find a workaround that worked from the terminal.
```
QT_STYLE_OVERRIDE=gtk2
export QT_STYLE_OVERRIDE
nixnote2
```
The nixnote GUI looks different, but at least it allowed me to get the authorisation from Evernote without the segfault. Then, after getting authorisation and exiting from nixnote, I managed to run the GUI as normal from the start menu shortcut.

This DOES work around the issue, and as it only affects the initial authorisation, it only needs to be done once - but I'm not sure we can rely on that in the long term. If anyone who understands how display managers interact with widgets in QT5 can shed any light, it might help avoid others suffering a bad initial experience with this great program. 
Thanks, Art
